tests/foreman/__init__.py

```diff
@@ -0,0 +1,3 @@
+from robottelo.config import settings
+
+settings.configure()
```

Comment:
This is currently not allowing generating documentation for the tests.foreman.* modules.